mcs/packages/Makefile

```diff
@@ -3,7 +3,7 @@ include ../build/rules.make
 
 ROSLYN_CSC_DIR = $(dir $(CSC_LOCATION))
 
-ROSLYN_FILES = \
+ROSLYN_FILES_FOR_MONO = \
```

Comment:
To sum up, I think we only need Microsoft.Build.Tasks.CodeAnalysis.dll, Microsoft.CSharp.Core.targets and Microsoft.VisualBasic.Core.targets in msbuild.

We'd need csi.*, Microsoft.CodeAnalysis.CSharp.Scripting.dll and Microsoft.CodeAnalysis.Scripting.dll in lib/mono/4.5 to support C# interactive (which would be mine and @marek-safar's task).